patternfly/patternfly-elements
# fix(icon): use icons package

## What I did

1. remove the existing icons build script
2. replace it with a script that copies over patternfly icons (mostly fontawesome 5)

## Testing Instructions

1. [DP](https://deploy-preview-2502--patternfly-elements.netlify.app/components/icon/demo/sets/)
2. copy locally into RHDS and check footer, navigation-secondary

## Notes to Reviewers

1. ~~pretty sure this is a breaking change. I think we can avoid a major release by restoring the old build script and running the new one on top of it, then any icons of the same file name can be considered 'fixes' without breaking the path they're stored at~~ breakage has been avoided
